File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/N. Char/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/N

#include <iostream>
using namespace std;

int main()
{
    char c;
    cin >> c;
    if (islower(c))
    {
        cout << char(c - 32) << endl;
    }
    else
    {
        cout << char(c + 32) << endl;
    }
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/N. Char/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/N

using System;

namespace Codeforces.Assuit.Sheet1
{
    class ProblemK
    {
        static void Main(string[] args)
        {
            char c = Convert.ToChar(Console.ReadLine());
            if (Char.IsLower(c))
            {
                Console.WriteLine((char)(c-32));
            }
            else
            {
                Console.WriteLine((char)(c+32));
            }
        }
    }
}
